bin/cluster/management_update

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/perl
+
+=head1 NAME
+
+Script to call when there is a change in the management state
+
+=head1 SYNOPSIS
+
+bin/cluster/management_update <change_type> <vrrp_ip> <state> <priority>
+
+=head1 DESCRIPTION
+
+Made for keepalived so the pf services can be adjusted depending of the running node
+
+=cut
+
+use constant INSTALL_DIR => '/usr/local/pf';
+use lib INSTALL_DIR . "/lib";
+
+use pf::log;
+
+my $logger = get_logger;
+
+$logger->info("Refreshing services for management mode");
+
+my $vrrp_ip = $ARGV[1];
+my $state = $ARGV[2];
+my $priority = $ARGV[3];
+
+if($state eq "MASTER"){
+    $logger->info("Transition to master state, starting management services.");
+    `/usr/local/pf/bin/pfcmd service pfdhcplistener start`;
```

Comment:
And then in the implementation I adapted, the master server is the master of all interfaces in keepalive so it will own all the registration interfaces. This means all pfdhcplistener is listening on all interfaces.